consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/BreathingSpaceServiceService.java

```diff
@@ -3,23 +3,25 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import uk.gov.moj.sdt.cmc.consumers.api.CMCApi;
-import uk.gov.moj.sdt.cmc.consumers.api.IBreathingSpace;
+import uk.gov.moj.sdt.cmc.consumers.api.IBreathingSpaceService;
 import uk.gov.moj.sdt.cmc.consumers.request.BreathingSpaceRequest;
 import uk.gov.moj.sdt.cmc.consumers.response.BreathingSpaceResponse;
 
 
 @Service("BreathingSpaceService")
-public class BreathingSpaceService implements IBreathingSpace {
+public class BreathingSpaceServiceService implements IBreathingSpaceService {
```

Comment:
Should this be named `BreathingSpaceService` rather than `BreathingSpaceServiceService`?

Reply:
refactoring issue, fixed it now